code/__DEFINES/misc.dm

```diff
@@ -38,6 +38,7 @@ Will print: "/mob/living/carbon/human/death" (you can optionally embed it in a s
 #define VALENTINES				"Valentine's Day"
```

Comment:
Sure, but defining it all here is a really better way to do it than hardcoding all the holidays names.
Also, as define is preprocessed, there should be no performance issues.

In other words, adding the holidays here is a good practice (the best practice is to create a JSON containing holiday names, descriptions and dates).